rewrite-gradle/src/main/java/org/openrewrite/gradle/RemoveDependency.java

```diff
@@ -78,6 +84,39 @@ protected TreeVisitor<?, ExecutionContext> getVisitor() {
             final MethodMatcher dependencyDsl = new MethodMatcher("DependencyHandlerSpec *(..)");
             final DependencyMatcher dependencyMatcher = requireNonNull(DependencyMatcher.build(groupId + ":" + artifactId).getValue());
 
+            @Override
+            public J visitJavaSourceFile(JavaSourceFile cu, ExecutionContext ctx) {
```

Comment:
We've gotten rid of `visitJavaSourceFile()` in rewrite 8.0.
This recipe will be easier to port if we go ahead and stop using it here, too. 

Can use `visitComilationUnit()`

Reply:
Done via 807efaca908ce525a97466f52e91f7206a4d1039.